src/components/MapView/Legends/index.tsx

```diff
@@ -398,6 +398,30 @@ function LegendItem({
               Download
             </Button>
           </Grid>
+
+          {exposure && !analysisResult && (
+            <AnalysisButton
+              variant="contained"
+              color="primary"
+              size="small"
+              onClick={runExposureAnalysis}
+            >
+              Exposure Analysis
+            </AnalysisButton>
+          )}
+
+          {exposure && analysisResult && (
+            <AnalysisButton
+              variant="contained"
+              color="secondary"
+              size="small"
+              onClick={() => dispatch(clearAnalysisResult())}
+            >
+              clear analysis
```

Comment:
```suggestion
              Clear Analysis
```
Nice to case these well even if they are in all caps just in case we revert back to Normal case ;)

Reply:
Good point. Done.